src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -547,8 +547,21 @@ public uint Size
 #else
         private async Task NixCancelAndKillProcessTree()
         {
-            // TODO: replace Task.Delay(1) with Send SIGINT/SIGTERM/SIGKILL
-            await Task.Delay(1);
+            bool sigint_succeed = await SendSignal(Signals.SIGINT, TimeSpan.FromSeconds(10));
```

Comment:
Can we have a descriptive constant here?  This is the graceful cancel max time, correct?